From: Thomas Gleixner <tglx@linutronix.de>
Date: Sat, 19 May 2018 16:38:59 +0200
Patch-mainline: v4.18-rc1
Subject: x86/Hyper-V/hv_apic: Include asm/apic.h
Git-commit: 61eeb1f6d1f2648a218855d7c8d44f16df242ef3
References: bsc#1107207

Not all configurations magically include asm/apic.h, but the Hyper-V code
requires it. Include it explicitely.

Fixes: 6b48cb5f8347 ("X86/Hyper-V: Enlighten APIC access")
Reported-by: kbuild test robot <lkp@intel.com>
Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Cc: K. Y. Srinivasan <kys@microsoft.com>
Cc: Michael Kelley <mikelley@microsoft.com>
Acked-by: Olaf Hering <ohering@suse.de>
---
 arch/x86/hyperv/hv_apic.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/hyperv/hv_apic.c b/arch/x86/hyperv/hv_apic.c
--- a/arch/x86/hyperv/hv_apic.c
+++ b/arch/x86/hyperv/hv_apic.c
@@ -29,6 +29,7 @@
 #include <linux/cpuhotplug.h>
 #include <asm/hypervisor.h>
 #include <asm/mshyperv.h>
+#include <asm/apic.h>
 
 #ifdef CONFIG_X86_64
 #if IS_ENABLED(CONFIG_HYPERV)
